install.ps1

```diff
@@ -1,50 +1,56 @@
-# param (
-#     [Parameter(Mandatory)] $PersonalToken  
-# )
+param (
+    # [Parameter(Mandatory)] $PersonalToken  
+    [Parameter(Mandatory = $false)] [System.Management.Automation.SemanticVersion]$DaggerVersion,
```

Comment:
I get this error when I run `install.ps1`:

```
Unable to find type [System.Management.Automation.SemanticVersion].
At C:\Users\Home\src\dagger\install.ps1:3 char:37
+ ... ory = $false)] [System.Management.Automation.SemanticVersion]$DaggerV ...
+                    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ CategoryInfo          : InvalidOperation: (System.Manageme...SemanticVersion:TypeName) [], RuntimeException
+ FullyQualifiedErrorId : TypeNotFound
```

Reply:
PS C:\Users\Home\src\dagger> $PSVersionTable
>>

Name                           Value
----                           -----
PSVersion                      5.1.22000.653
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.22000.653
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1

Reply:
Looks like I need to move to PS 7.x. 😄 
https://github.com/PowerShell/PowerShell/issues/16330

Reply:
There are likely other folks on old Powershell versions. Can we test for and require a minimum PS version in the script so folks know what to do?

Reply:
Trying this: `winget install Microsoft.PowerShell`

<img width="1252" alt="Screen Shot 2022-08-12 at 1 41 45 PM" src="https://user-images.githubusercontent.com/3187222/184442081-a24639aa-410a-48d2-a5da-d97536f1fef9.png">

Reply:
<img width="976" alt="image" src="https://user-images.githubusercontent.com/3187222/184442975-2d9fc2ec-b80b-45ef-b565-0eeeb2c52204.png">

`-Interactive $true` asked me to proceed, but default it's set to `$false` so just installs. Could easily be flipped if important, so 👍 
<img width="1100" alt="image" src="https://user-images.githubusercontent.com/3187222/184443093-43cc6672-ec89-4643-81d7-f3d0d4813012.png">

Version and location settings work perfectly.

Reply:
So the summary is, works great, but we need to let folks know to be on `pwsh` aka version 7.x.
Thanks for this contribution @brittandeyoung 
It would be ideal to add requirement of `pwsh` version to the doc for sure, and to the script if possible.

Reply:
Powershell version requirement added to the script and docs.